modules/ocf_www/files/lets-encrypt-update

```diff
@@ -254,24 +95,27 @@ def main(argv=None):
         global debug
         debug = functools.partial(print, flush=True)
 
-    eligible = eligible_domains(args.target)
-    have_cert = set()
-    need_cert = set()
-    for domain in eligible:
-        assert re.match('^[a-z\-_\.0-9]+$', domain)
-        cert_path = CERT_STORAGE_DIR / '{}.crt'.format(domain)
-
-        if cert_path.exists():
-            have_cert.add(domain)
-            maybe_renew(domain, cert_path, dry_run=args.dry_run)
-        else:
-            need_cert.add(domain)
-            maybe_acquire(domain, cert_path, dry_run=args.dry_run)
+    if args.target == 'web':
+        vhosts, target_domain = get_vhosts(), 'death.ocf.berkeley.edu'
+    else:
+        vhosts, target_domain = get_app_vhosts(), 'werewolves.ocf.berkeley.edu'
+
+    domains = all_domains(vhosts)
+    eligible = eligible_domains(domains, target_domain)
+
+    ret = subprocess.call(chain(
```

Comment:
Somebody's been coding in Haskell a lot, I can tell :)

Why don't we try making use of PEP 448...

Reply:
Good suggestion, but this has to run on Python 3.4 because death is still on jessie. Also, this doesn't seem particularly haskell-esque to me.